metadata-ingestion/tests/integration/cassandra/docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: "1"
+services:
+  test-cassandra:
+    image: cassandra:latest
+    container_name: test-cassandra
+    ports:
+      - 9042:9042
+    volumes:
+      - ./setup/cassandra.yaml:/etc/cassandra/cassandra.yaml
+      - ./setup/init_keyspaces.cql:/docker-entrypoint-initdb.d/init_keyspaces.cql
```

Comment:
Just for my own understanding, do we need volume mount of init_keyspaces here given that the data is loaded by another container and not this one ?